contact.html

```diff
@@ -27,7 +27,7 @@ <h1><a href="index.html">BCIT</a></h1>
 
    <div class="wrapper">
        <main>
-            <a name="top"></a>
+            <a href="#"></a>
```

Comment:
What is this for?

Reply:
It is showing where the top of the page, so when you push button back to top, you going to this level. Validator showed that "a name" is obsolete, so I changed it. Now it is valid.